exporter.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import logging
+from openpyxl import Workbook
+import time
```

Comment:
`logging` and `time` are from the standard library so should be in the top block